src/mainboard/seeed/beaglev/DIT

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
```

Comment:
This should probably be in the Makefile.

Reply:
yep.